docs/reference/resource-profiling.md

```diff
@@ -9,12 +9,12 @@ The results are summarized as follows:
 
 | Components |  Processor | Min CPU | Min RAM with Kine/SQLite | Min RAM with Embedded etcd |
 |------------|-----|----------|-------------------------|---------------------------|
-| K3s server with a workload  | Intel(R) Xeon(R) Platinum 8124M CPU, 3.00 GHz | 10% of a core | 768 M | 896 M |
-| K3s cluster with a single agent  | Intel(R) Xeon(R) Platinum 8124M CPU, 3.00 GHz | 10% of a core | 512 M | 768 M |
-| K3s agent | Intel(R) Xeon(R) Platinum 8124M CPU, 3.00 GHz | 5% of a core | 256 M | 256 M |
-| K3s server with a workload  | Pi4B BCM2711, 1.50 GHz | 20% of a core | 768 M | 896 M |
-| K3s cluster with a single agent | Pi4B BCM2711, 1.50 GHz | 20% of a core | 512 M | 768 M |
-| K3s agent  | Pi4B BCM2711, 1.50 GHz | 10% of a core | 256 M | 256 M |
+| K3s server with a workload  | Intel 8375C CPU, 2.90 GHz | 6% of a core | 1602 M | 1604 M |
+| K3s cluster with a single agent  | Intel 8375C CPU, 2.90 GHz | 5% of a core | 1428 M | 1450 M |
```

Comment:
The memory utilization for kine was within 2M of etcd for both test cases? That is highly unexpected and seems unusual; etcd usually uses a noticeable chunk more memory.

Reply:
Retested 1 Server workload:
kine before workload: 1419M, after: 1596M
etcd before workload: 1425M, after: 1606M

On the x86 arch there is not that much difference between the DBs. I note that this is always "at rest" after installation/addition of the workload. There is a consistent garbage collection pass that tends to drop the memory usage after about 1 min. Its possible that kine/sqlite has less usage during spikes or in more active clusters.